dissect/target/plugins/apps/ssh/openssh.py

```diff
@@ -190,3 +194,36 @@ def parse_known_host(known_host_string: str) -> tuple[str, list, str, str, str]:
     comment = " ".join(parts[3:]) if len(parts) > 3 else ""
 
     return marker, hostnames.split(","), keytype, public_key, comment
+
+
+def calculate_fingerprints(public_key_decoded: bytes, ssh_keygen_format: bool = False) -> tuple[str, str, str]:
```

Comment:
Maybe this function is better placed in the `ssh.py` file? Since you also use it in `putty.py`.